.env.example

```
@@ -0,0 +1,8 @@
+# PostgreSQL
+DATABASE_URL="postgresql://postgres:postgres@localhost:5432/devin_task_board"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)